code/game/objects/items/weapons/grenades/firecracker.dm

```diff
@@ -0,0 +1,88 @@
+/obj/item/grenade/firecracker
+	name = "fire-cracker"
+	desc = "A firework that's great at making a lot of noise."
+	det_time = 2 SECONDS
+	/// The effect that will be randomly played at intervals.
+	var/sound_effect = 'sound/weapons/gunshots/gunshot_strong.ogg'
```

Comment:
We probably don't want admins to be able to change the sound of these firecracks, it could be a define. Not needed to be changed though, just a note.

Reply:
Honestly we do, this could allow for some funny adminbus